docs/tarball-readme.md

```diff
@@ -0,0 +1,11 @@
+# OpenTelemetry Collector for Google Cloud Monitoring Tarball
+
+Thank you for using this OpenTelemetry Collector and Google Cloud Monitoring. This packaged tarball contains tools to immediately setup the Collector with compatible metrics for your Google Cloud Monitoring.
```

Comment:
Since we are not really going to be "officially" supporting this, can you say "Thank you for using this community build of the OpenTelemetry Collector for Google Cloud Monitoring..."

Reply:
> "This packaged tarball contains tools to immediately setup the Collector with compatible metrics for your Google Cloud Monitoring."

I'm not really sure what that part means. You also need to describe what these metrics are / where they come from.

Reply:
Let me know how it looks now.

Reply:
It's the same isn't it? Did you forget to commit something

Reply:
It's the same isn't it. Did you forget to commit something?

Reply:
Sorry, don't know what happened there. It is updated now :)